src/controller/python/test/test_scripts/cluster_objects.py

```diff
@@ -319,6 +321,9 @@ async def TestReadEventRequests(cls, devCtrl, expectEventsNum):
 
     @classmethod
     async def TestTimedRequest(cls, devCtrl):
+        if not TestIsEnabled("datamodel.timedrequest"):
```

Comment:
Why not just follow a naming convention that matches the test method name? That'd make it easier to match up.

Reply:
Changed the name, will use Python's reflect features (like Python's unit_test) in separate pr.